Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,135 @@ export default function PPC_MPPT(props) {
                     <Spacer/>
                     <Flex w='45%' direction='column'>
                         <MPPT_CELL
-                            label='Charge'
+                            label='Tracking'
                             isCurrMode={props.data?.mppt_mode[0]}
                         />
                         <Spacer/>
                         <MPPT_CELL
-                            label='MPPT'
+                            label='Current Control'
                             isCurrMode={!props.data?.mppt_mode[0]}
                         />
```

Comment:
Two things:
1. We should add an optional `fontSize` (and maybe/probably `lineHeight`) prop(s) to `MPPT_CELL` so that we can decrease the font size (and maybe/probably line height) of the text in the component. Currently, the text for this `MPPT_CELL` is a bit big and is straddling the borders of the component.
2. The `Text` component in `MPPT_CELL` should center align/justify its text so that wrapping text is centered rather than left justified.